spec/Section 5 -- Validation.md

```diff
@@ -1173,8 +1173,8 @@ declaration, not its body.
 ##### Object Spreads In Abstract Scope
 
 Union or interface spreads can be used within the context of an object type
-fragment, but only if the object type is one of the possible types of
-that interface or union.
+fragment or interface type fragment, but only if the type is one of the
```

Comment:
I think this change is misplaced (maybe should go into "Abstract Spreads in Abstract Scope" below)